test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2310,6 +2310,93 @@ def forward(self, x, y):
         y = torch.rand(100)
         self.run_test(PowModule3(), (x, y))
 
+    # the arithmeticOps(Add\Sub\Mul\Div\Gemm\Pow\Mod) with low precision include unit8 will be failed in ORT
+    # add to(dtype=torch.long) to avoid ORT output type does not match expected type
+    def test_arithmeticOps_with_low_precision(self):
+        class AddModule(torch.nn.Module):
+            def forward(self, x, y):
+                return (x + y).to(dtype=torch.long)
+
+        class SubModule(torch.nn.Module):
+            def forward(self, x, y):
+                return (x - y).to(dtype=torch.long)
+
+        class MulModule(torch.nn.Module):
+            def forward(self, x, y):
+                return (x * y).to(dtype=torch.long)
+
+        class DivModule(torch.nn.Module):
+            def forward(self, x, y):
+                return (x / y).to(dtype=torch.long)
+
+        class PowModule(torch.nn.Module):
+            def forward(self, x, y):
+                return x.pow(y).to(dtype=torch.long)
+
+        x = torch.tensor([2, 3, 5], dtype=torch.uint8)
```

Comment:
Can you add a test for adding uint8 with unin16? And uint8 with uint32? Just to check type promotions.